plugins/com.google.cloud.tools.eclipse.appengine.localserver/META-INF/MANIFEST.MF

```diff
@@ -9,9 +9,9 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Activator: com.google.cloud.tools.eclipse.appengine.localserver.Activator
 Bundle-ActivationPolicy: lazy
 Bundle-Localization: plugin
-Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.6.6",
+Require-Bundle: com.google.cloud.tools.appengine;bundle-version="[0.6.8,0.7.0)",
```

Comment:
Does specifying 0.6.8 not prevent higher versions?

Reply:
It's the equivalent of "[0.6.8,∞]"